components/board.drugconnectivity/R/drugconnectivity_plot_cmap_dsea.R

```diff
@@ -0,0 +1,298 @@
+##
+## This file is part of the Omics Playground project.
+## Copyright (c) 2018-2022 BigOmics Analytics Sagl. All rights reserved.
+##
+
+#' Drug Connectivity plot UI input function
+#'
+#' @description A shiny Module for plotting (UI code).
+#'
+#' @param id
+#' @param label
+#' @param height
+#'
+#' @export
+drugconnectivity_plot_cmap_dsea_ui <- function(id,
+                                               label = "",
+                                               height = c(750, 750)) {
+  ns <- shiny::NS(id)
+  info_text <- strwrap("<strong>Connectivity map.</strong> correlates your
+                       signature with known drug profiles from the L1000
+                       database, and shows similar and opposite profiles by
+                       running the GSEA algorithm on the drug profile
+                       correlation space.")
+
+  plot_opts <- shiny::tagList(
+    tipifyL(
+      shiny::radioButtons(
+        ns("cmap_labeltype"), "label type:",
+        c("drugs", "MOA", "target"),
+        inline = TRUE
+      ),
+      "Label point with drugs, MOA terms or targets (if no drug selected)."
+    ),
+    tipifyL(
+      shiny::radioButtons(ns("cmap_nlabel"), "number of labels:", c(3, 10, 20, 100),
+        selected = 10, inline = TRUE
+      ),
+      "Number of labels to show."
+    ),
+    tipifyL(shiny::checkboxGroupInput(
+      ns("cmap_labeloptions"), "label options:",
+      choices = c("show", "fixed"),
+      selected = c("show"), inline = TRUE
+    ), "Other labels options.")
+  )
+
+  PlotModuleUI(ns("plot"),
+    title = "Connectivity Map",
+    label = label,
+    plotlib = "base",
+    info.text = info_text,
+    options = plot_opts,
+    download.fmt = c("png", "pdf", "csv"),
+    width = c("auto", "100%"),
```

Comment:
Same as previous file, although here I see the `height` matches, so I assume that you have changed the `width` to be better aesthetically?